app/resources/card_resource.rb

```diff
@@ -163,16 +163,16 @@ class CardResource < ApplicationResource # rubocop:disable Metrics/ClassLength
 
   private
 
-  def images(id, has_narrative_image: false, face_index: nil)
+  def images(id, designed_by, has_narrative_image: false, face_index: nil)
```

Comment:
if we are going by set, we want released_by, not designed_by.

For example. the latest printing of sure gamble does not have xlarge, but the class act does right now:

<img width="798" height="206" alt="Image" src="https://github.com/user-attachments/assets/241ada00-c6c4-4742-b8ca-20561cbb38db" />
<img width="741" height="233" alt="Image" src="https://github.com/user-attachments/assets/2367aaf1-04f4-4c9e-938e-745c8c32f326" />